src/process_recipes.jl

```diff
@@ -1,5 +1,6 @@
 
 _preprocess_args(p, args, s) = args #needs to modify still_to_process
```

Comment:
Should this be named `_preprocess_args!`?

Reply:
yes

Reply:
depends on whether it should return the processed args or mutate them inplace

Reply:
It mutates `p` and `s` in Plots.

Reply:
and also, the default implementation should change `s`, or else the `_process_plotrecipe` is not reached